easybuild/easyblocks/n/netcdf.py

```diff
@@ -71,6 +71,27 @@ def configure_step(self):
 
             CMakeMake.configure_step(self)
 
+        if LooseVersion(self.version) >= LooseVersion("4.3.3"):
+            set.cfg.update('configopts', '-DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_C_FLAGS_RELEASE="-DNDEBUG " ')
+            hdf5 = get_software_root('HDF5')
+            hdf5_libdir = get_software_libdir('HDF5')
+            szip_libdir = get_software_libdir('Szip')
+            if hdf5_libdir and szip_libdir:
+                set.cfg.update('configopts', '-DHDF5_LIB=%s/libhdf5.so ' % hdf5_libdir)
+                set.cfg.update('configopts', '-DHDF5_HL_LIB=%s/libhdf5_hl.so ' % hdf5_libdir)
+                set.cfg.update('configopts', '-DHDF5_INCLUDE_DIR=%s/include ' % hdf5)
+                set.cfg.update('configopts', '-DSZIP_LIBRARY=%s/libsz.so ' % szip_libdir)
```

Comment:
See ZLIB. I think szip and zlib just deps of other libs (HDF5?) so include dirs are not needed,